src/templates/preview.html

```diff
@@ -1,7 +1,12 @@
 <div id="animated-preview-container" class="preview-container">
   <div class="canvas-container-wrapper minimap-container">
     <div class="preview-contextual-actions">
-      <div class="preview-contextual-action  original-size-button" rel="tooltip" data-placement="bottom">1x</div>
+      <div class="preview-drop-down">
+        <div class="preview-disable-overlay" rel="tooltip" data-placement="left"></div>
+        <div class="preview-contextual-action  original-size-button" rel="tooltip" data-placement="left">1x</div>
+        <div class="preview-contextual-action  best-size-button" rel="tooltip" data-placement="left"></div>
+        <div class="preview-contextual-action  full-size-button" rel="tooltip" data-placement="left">Full</div>
```

Comment:
I don't mind either way, and the performance difference should not be noticeable here. I tend to leave it in the markup if I don't have to create similar elements dynamically. 

In this case I don't think you'll be creating elements on the fly, so I would probably just leave it in the markup. But if it's easier for you to generate it dynamically, go ahead.